subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -16,6 +16,11 @@
 
 LOG_MODULE_REGISTER(aws_fota, CONFIG_AWS_FOTA_LOG_LEVEL);
 
+/* State variable to ensure that it does not update the job execution document
```

Comment:
What does "it" refer to?

Reply:
The library itself. Agree that it's ambiguous will change it to AWS Fota library